### ISSUE TITLE
Error installing MaterialComponents

### ISSUE THREAD

**Issue by huyaxiong** [NativeScript/nativescript-cli#5279]
**Environment**
Provide version numbers for the following components (information can be retrieved by running `tns info` in your project folder or by inspecting the `package.json` of the project):
✔ Getting NativeScript components versions information...
✔ Component nativescript has 6.4.1 version and is up to date.
✔ Component tns-core-modules has 6.4.2 version and is up to date.
⚠ Update available for component tns-android. Your current version is 6.0.0 and the latest available version is 6.4.1.
⚠ Update available for component tns-ios. Your current version is 6.0.1 and the latest available version is 6.4.2.

**Describe the bug**
[!] Error installing MaterialComponents
[!] /usr/bin/git clone https://github.com/material-components/material-components-ios.git /var/folders/h3/v3ty877j36bgkn4mr81hzdkm0000gp/T/d20200311-61109-m69iti --template= --single-branch --depth 1 --branch v94.5.0

Cloning into '/var/folders/h3/v3ty877j36bgkn4mr81hzdkm0000gp/T/d20200311-61109-m69iti'...
error: RPC failed; curl 56 LibreSSL SSL_read: SSL_ERROR_SYSCALL, errno 54
fatal: the remote end hung up unexpectedly
fatal: early EOF
fatal: index-pack failed

'pod install' command failed.
**To Reproduce**
After creating a new project, and then run command 'tns run ios', the error above occured.

**Expected behavior**
I can preview the newly created sample project on the simulator.

**Sample project**

**Additional context**

**Comment by NickIliev** [comment]
@huyaxiong 
```
Update available for component tns-ios. Your current version is 6.0.1 and the latest available version is 6.4.2.
```
So update the runtime with
```
tns platform remove ios
tns platform add ios@latest
```

 And also run `pod repo update`

**Comment by huyaxiong** [comment]
I already switched to another tool and currently, I dont have time to verify if it will work, so I will close this issue for now, thank u for your reply.
